apps/medon-fe/src/components/Dashboard/index.tsx

```diff
@@ -2,28 +2,44 @@ import { useTranslation } from 'react-i18next';
 import { getUserSelector } from 'redux/features/userSlice/userSelectors';
 import { useAppSelector } from 'redux/hooks';
 
-import { Container } from 'components/Dashboard/styles';
+import { AppointmentContainer, Container } from 'components/Dashboard/styles';
 import WithoutAppointments from 'components/WithoutAppointments';
 import Attention from 'components/common/Attention';
 import AppointmentsScore from 'components/AppointmentsScore';
-import AppointmentsList from 'components/AppointmentsList';
-import { patientList } from 'utils/mock/patientList';
+import { AppointmentsCard } from 'components/AppointmentsCard';
+
+import { appointmentCardMock } from 'utils/mock/appointment';
 
 export default function Dashboard() {
   const user = useAppSelector(getUserSelector);
   const { t } = useTranslation();
-  const hasAppointments = patientList && patientList.length > 0;
 
   return (
     <Container>
       <h1>
-        {t('dashboard.welcome')}, Dr. {`${user.lastName || 'Anonymous'}`}
+        {t('dashboard.welcome')}, {t('dashboard.prefix-doctor')}
+        {`${user.lastName || 'Anonymous'}`}
       </h1>
-      {hasAppointments ? (
+      {appointmentCardMock.length ? (
         <>
           <Attention />
-          <AppointmentsScore />
-          <AppointmentsList />
+          <AppointmentsScore quantity={appointmentCardMock.length} />
+          <AppointmentContainer>
+            {appointmentCardMock
```

Comment:
I dont like that you will reorginice it with each rerender, its expensive operators

Reply:
okay, fixed